assets/Gruntfile.js

```diff
@@ -207,6 +208,8 @@ module.exports = function (grunt) {
           'bower_components/uri.js/src/URI.min.js',
           'bower_components/uri.js/src/jquery.URI.min.js',
           'bower_components/uri.js/src/URI.fragmentQuery.js',
+          // bower registration error? we get 2x versions of uri.js/urijs
+          'bower_components/urijs/',
```

Comment:
Just confirming, you are seeing it as well?  I believe it appears even when I'm on `console_1.3` branch.  I'll check it again.